app.js

```diff
@@ -199,7 +199,7 @@ If you encounter any issue/typo visit https://github.com/Marr11317/muscord-bot.
         if (containsOneIn(msg.content, ["Thank you", "Merci", "Danke", "Gracias"]) && msg.author.id !== client.user.id) {
             send(msg.channel, ["You're Welcome", "It's a pleasure", "Wow that's kind", ":heart:", "De nada", "El placer es mio", "De rien", "Je suis la pour vous servir..."]);
         }
-        if (containsOneIn(msg.content, ["musebot"])) {
+        if (containsOneIn(msg.content, ["@musebot"])) {
```

Comment:
same here. Why would anyone say musebot? It will almost never happen, and @musebot will litterally never happen. But again, it might only be funny the first times.

Reply:
Oh right I just saw your comment.